cmd/version.go

```diff
@@ -0,0 +1,25 @@
+package cmd
+
+import (
+	"io"
+	"os"
+
+	"github.com/konstructio/dropkick/configs"
+	"github.com/konstructio/dropkick/internal/logger"
+	"github.com/spf13/cobra"
+)
+
+func getVersionCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "version",
```

Comment:
Besides creating the `version`, you should also specify the version in the root command's `Version` attribute:

https://github.com/spf13/cobra/blob/v1.8.1/command.go#L108-L112

Technically, you don't need a `version` command since by default cobra will register `-v` as a version (and not as "verbose").

Reply:
<img width="596" alt="image" src="https://github.com/user-attachments/assets/6b327df8-4389-4ec2-9be2-9adcfa2859ed">
so we would specify it to produce this and we can support both?

Reply:
That's up to us. If you want to make it just be `dropkick version`, then you can let the root know that `-v` is no longer necessary.

Reply:
i prefer the `dropkick version` command to the `-v` variation. ill drop it from rootCmd if you're fine with that

Reply:
Sounds good! 

For reference for the future, POSIX does not define subcommands for versions although they're still widely used. The goal is that in POSIX subcommands become "extensions" while "version" is a property of the CLI, that's why Cobra follows that stance instead of the subcommand.